mock_awx/settings/__init__.py

```diff
@@ -12,15 +12,13 @@
     }
 }
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
-INSTALLED_APPS = ['awx.conf']
```

Comment:
we may need to keep `INSTALLED_APPS = []`, I suspect the test failures could be related to that.